### ISSUE TITLE
Could there be a csvz *folder* without zip?

### ISSUE THREAD

**Issue by doekman** [secretGeek/csvz#15]
When you unzip a `csvz` file, you end up with a folder with csv-files.

Can this be considered a csvz-container?  
I think it could be useful. For example, when putting datafiles into `git`.

To differentiate a csvz-container from a folder containing some csv files, I propose such a folder to have the `.csvd` extension. So if you extract the `my_data.csvz` file, you get the `my_data.csvd` folder. 

The folder could have the same extension (thus not introducing a new extension). The disadvantage is you can't extract an `.csvz` file into the same folder without deleting/moving the original file.

Tools are not expected to open these `.csvd`-folders directly. It's only to denote that when zipped, it's automatically a `.csvz` file.

**Comment by secretGeek** [comment]
Yes definitely. Folder that complies with all the other spec fragments but just happens to not
comply with `csvz-0` is still a csvz. (It just doesn’t comply with that one fragment, csvz-0)

--Most-- (note doeke corrects this in next comment, it should say "some" not "most") tools in this space would probably only deal with unpacked csvz files (ie folders full of files... not zips) and would leave the packing or unpacking up to a special tool (ie a compression tool). This is what I thought after asking about tar, gzip and just generally thinking about the Unix way.

**Comment by secretGeek** [comment]
Todo: make this clear in the spec. Then this one can be closed.

**Comment by doekman** [comment]
One note: I think most tool will unzip to memory, that's how docx work. It's quicker: disk reads are slower than unzipping.
